### DISCUSSION TITLE
Minimap headers not detected in language (bug?)

### DISCUSSION THREAD

**Discussion by elringus** [microsoft/vscode-discussions#1133] [Extension Development QnA] [answered]
Not sure if that's a bug or I'm just missing some config, but the new minimap headers (https://github.com/microsoft/vscode/pull/190759) are not detected with my language configuration.

I have comments specified in the [lang-config.json](https://github.com/naninovel/vscode/blob/1.20/language-configuration.json) as follows:

```json
{
    "comments": {
        "lineComment": ";"
    },
}
```

But it doesn't work:

![](https://i.gyazo.com/c3682c19a9a357a81c0a21baef27d0bc.png)

Here are the full sources of the language extension: https://github.com/naninovel/vscode/tree/1.20

Would appreciate any ideas on what I may be missing or if this should be reported as a bug.

**Comment by abrilurena** [comment]
Hi, @elringus! I'm adding @isidorn he might be able to help

**Reply by isidorn** [reply]
I suggest to report this as a bug, or comment on the PR you linked. Thank you

**Reply by elringus** [reply, marked as answer]
Thanks, I've created a a bug report: https://github.com/microsoft/vscode/issues/210894

**Reply by burkeholland** [reply]
Marking this as the answer. If you feel this is an error, please feel free to re-open.
